zapier/introduction/zaps-ii/zapier-search-and-create.md

```diff
@@ -0,0 +1,36 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Search and Create
+
+---
+## Content
+
+Depending on the app that you are using, Zapier allows you to create date after doing a search action.
+
+A good workflow example would be to monitor some data, and when a certain condition is met you create a new record/data.
+
+Let's take another look at the zap we have created. After you receive a message that a person was given access to your base, you want to write a new cell that specifies the number of access days.
+
+Go ahead and add another Google Sheets step to our zap, right at the end of the flow. This time, we want to create a new column, so you would have to search the `'Choose Action Event'` drop-down for the `'Create Spreadsheet Column'` option.
```

Comment:
so this will only create the column but not fill in values. the latter will be discussed in a later insight, right? if so, this is worth mentioning to avoid confusion